Fix: text underline on the Login & Sign Up (blue) buttons

sachinsom93 opened the pull request:
### Description

Removed the unnecessary underlines under login and sign Up buttons

Fixes #177 and replaces pr #190

### Type of Change:
<!--**Delete irrelevant options.**-->

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
![Register page](https://github.com/sachinsom93/temp/blob/master/register.jpg?raw=true)
![Login page](https://github.com/sachinsom93/temp/blob/master/Login.jpg?raw=true)


### Checklist:
<!--**Delete irrelevant options.**-->

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

decon-harsh commented:
Tested locally, LGTM @sachinsom93  

![Screenshot from 2021-04-17 12-51-45](https://user-images.githubusercontent.com/58214248/115105379-26c2e500-9f7c-11eb-8963-0a1931e90293.png)

![Screenshot from 2021-04-17 12-51-42](https://user-images.githubusercontent.com/58214248/115105380-27f41200-9f7c-11eb-9744-f99b67678a08.png)

sachinsom93 committed:
Type: Bug Fix (ISSUE #177)
Body:
    1. refactored login and signup buttun.
    2. Changed anchor tags to link tags

mtreacy002 reviewed (changes requested):
Great job on the fix, @sachinsom93 🎉🎉🎉. I just made minor change in the PR description by adding the old pr you've closed as it helps to know that others have given their reviews and suggestions on the issue so we can see how the process has evolved from there to here. I'm approving this PR now. 
Thank you again for your contribution 🎉🎉🎉.

Sorry, accidentally selected the wrong radio button 😊.
